app/containers/HouseScreen.js

```diff
@@ -6,7 +6,7 @@ import Spinner from 'react-native-loading-spinner-overlay';
 
 import ContainerList from "../components/ListComponent";
 import AvatarListItem from '../components/AvatarListItem';
-import { getAllHouses } from '../actions/index';
+import { getAllHouses, setHouse } from '../actions/index';
 
 class HouseScreen extends React.Component {
     state = {
```

Comment:
We may remove the empty `state`

Reply:
Its not empty state, please check the code

Reply:
Yep my mistake. Code section was collapsed.